hphp/hack/test/quickfixes/add_missing_awaitable_type.php.exp

```diff
@@ -0,0 +1,5 @@
+<?hh
```

Comment:
Sorry, missed this in my first review. Don't forget to run the tests!

The .exp files print the quickfix name, as well as the result. That helps us ensure the quickfix name is sensible. Something like this:

```suggestion
Make return type Awaitable
Change to `Awaitable<void>`
<?hh
```